### PR TITLE
Derek Cruz | Es6 template literals

### PR DESCRIPTION
**Pull request by Derek-Cruz**
es6-template-literals

### CODE REVIEW

File: es6-template-literals/index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+<body>
+  <script>
+    const number = 9200;
+    const street = 'Irvine Center Drive';
+    const address = `LearningFuze: ${number} ${street}`;
+
+    console.log(address);
```
Comment (TimelikeClosure) [inline es6-template-literals/index.html]: Please make sure you are A) doing one-to-one conversions of concatenations to template literals, and B) not modifying other parts of the code.

---

image

---

image

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Comment by Derek-Cruz** [comment]
es6-template-literals

File: es6-template-literals/index.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+<body>
+  <script>
+    const number = 9200;
+    const street = 'Irvine Center Drive';
+    const address = `${number} ${street}`;
+
+    console.log('LearningFuze:', address);
```
Comment (TimelikeClosure) [inline es6-template-literals/index.html]: Still looks like other parts of the code were modified, and the conversion wasn't exactly matched:

---

image

---

image

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Comment by Derek-Cruz** [comment]
es6-template-literals

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**